azure-pipelines.yml

```diff
@@ -187,9 +187,6 @@ jobs:
     - script: |
         brew upgrade
         brew install ccache flex bison
-        brew unlink git
-        brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/1024ec738fe0972e467859ce55687e9e0131f83a/Formula/git.rb
```

Comment:
This seems weird. Why not `brew install git`

Reply:
Please note these lines are being removed. This is from an earlier PR that intentionally added an older version of git to get around the issues that are fixed with a different strategy in this PR.

Reply:
Ha. Indeed. 💯